cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -188,28 +191,65 @@ public void sendMetric(final Metric metric) {
         }
     }
 
-    private User getIntercomUser(final MetricUser user) {
+    @Override
+    public MetricUser getUser(final String userId) {
+        if (userId == null) {
+            return null;
+        }
+        User intercomUser = null;
+        try {
+            intercomUser = User.find(userId);
+        } catch (final Exception e) {
+            logger.warn("Failed to get user with Id: {} from Intercom - {}", userId, e.getMessage());
+        }
+        if (intercomUser == null) {
+            return null;
+        } else {
+            return new MetricUser(intercomUser.getId(), getUserOrganisations(intercomUser), intercomUser.getName(),
+                    intercomUser.getEmail(), getCustomAttributes(intercomUser));
+        }
+
+    }
+
+    private User createIntercomUser(final MetricUser user) {
         if (user == null) {
             return null;
         }
 
         final User intercomUser = new User();
-        intercomUser.setUserId(user.id());
+        intercomUser.setId(user.id());
         intercomUser.setName(user.name());
 
         if (!Equals.isNullOrBlank(user.emailAddress())) {
             intercomUser.setEmail(user.emailAddress());
         }
 
-        if (!Equals.isNullOrBlank(user.organisationId())) {
-            final Company company = new Company();
-            company.setCompanyID(user.organisationId());
-            intercomUser.addCompany(company);
+        if (!user.organisationIds().isEmpty()) {
```

Comment:
This check is redundant. If the collection is empty, the `forEach` below will not have any effect.